.ci/jenkins/Jenkinsfile.buildchain

```
@@ -0,0 +1,111 @@
+@Library('jenkins-pipeline-shared-libraries')_
+
+additionalTimeout = "${env.ADDITIONAL_TIMEOUT?.trim() ?: 120}"
+
+agentLabel = "${env.ADDITIONAL_LABEL?.trim() ? ADDITIONAL_LABEL : 'kie-rhel7 && kie-mem16g'} && !master"
+build_jdk_tool = env.BUILD_JDK_TOOL?.trim() ?: 'kie-jdk11'
+build_maven_tool = env.BUILD_MAVEN_TOOL?.trim() ?: 'kie-maven-3.8.1'
+
+pr_type = env.BUILDCHAIN_PR_TYPE?.trim() ?: 'pr'
+build_chain_additional_arguments = "${env.BUILDCHAIN_PROJECT ? "-sp=${env.BUILDCHAIN_PROJECT?.trim()}" : ''}"
```

should be enough or ?
```suggestion
build_chain_additional_arguments = env.BUILDCHAIN_PROJECT ? "-sp=${env.BUILDCHAIN_PROJECT?.trim()}" : ''
```

---

well the idea was to be able to concatenate several env variables (in the future), your proposal is good for now... thanks

---

then we may use a list of string to gather the arguments and then use the `.join(' ')` method. wdyt ?